plugins/package-managers/maven/src/main/kotlin/utils/DependencyTreeLogger.kt

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
> The newly introduced functions are able to parse the output format of this goal

I'm not sure I'm ok with this. All ORT analyzers so far have abstained from parsing command output that is not meant for machine consumption. It's simply too fragile and error prone.

The [tree mojo](https://maven.apache.org/plugins/maven-dependency-plugin/tree-mojo.html) also seems to support [JSON output](https://maven.apache.org/plugins/maven-dependency-plugin/tree-mojo.html#outputtype), so why not used that instead?